### DISCUSSION TITLE
manually show Edit dialog of table?

### DISCUSSION THREAD

**Discussion by Morteza8181** [dotnetcore/BootstrapBlazor#4688] [Q&A]
How can I Manually call Table OnEdit to Show Edit dialog?
something like this:
        private Table<Foo> Table { get; set; }
        private async void OnTriggerEdit()
        {
            await Table.OnEditAsync(items[0]);
        }
this code won't work.
thank you!

**Comment by ArgoZhang** [comment]
@Morteza8181 sorry for the delay reply. for now there is noway to show the edit dialog.
